content/documentation/staging/tutorial/06-secure/01-traffic-policies.adoc

```diff
@@ -0,0 +1,128 @@
+---
+title: Authorization Policies
+weight: 56
+---
+:icons: font
+:sectlinks:
+
+link:https://istio.io/latest/docs/concepts/security/[Security, window="_blank"] is one of the main pillar of Istio features.
+
+The Istio https://istio.io/latest/docs/concepts/security/#high-level-architecture[Security High Level Architecture] provides a comprehensive solution to design and implement multiple security scenarios.
+
+In the context of this tutorial we are going to show how Kiali can combine telemetry information to create security policies for the workloads deployed on a given namespace.
```

Comment:
--> In this tutorial we will show how Kiali can use telemetry information to create security policies for the workloads deployed in a given namespace.